streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorStateManager.java

```diff
@@ -147,122 +187,124 @@ public File baseDir() {
     }
 
     @Override
-    public void register(final StateStore store,
-                         final StateRestoreCallback stateRestoreCallback) {
+    public void registerStore(final StateStore store, final StateRestoreCallback stateRestoreCallback) {
```

Comment:
If store is `null` then the first line `store.name()` would throw NPE directly, would that achieve the same?

Reply:
Well this is a good q, essentially they are the same effect, I would put this just as a nit then, because I could think of naming the culprit in the assertion of the passed in store.